UI/GameSettingsScreen.cpp

```diff
@@ -335,6 +336,11 @@ void GameSettingsScreen::CreateViews() {
 	altSpeed2->SetZeroLabel(gr->T("Unlimited"));
 	altSpeed2->SetNegativeDisable(gr->T("Disabled"));
 
+	if (KeyMap::AxisFromPspButton(VIRTKEY_SPEED_ANALOG, nullptr, nullptr, nullptr)) {
```

Comment:
This doesn't update when you get back from control mapping. If you exit to main menu and go back to setting screen it work tho'. Probably missing some `RecreateView()`.

Reply:
Yep, you're right.  Made it recreate.  I had tested with it already mapped by the time I changed this...

-[Unknown]